config/jobs/kubernetes/sig-testing/kubernetes-kind.yaml

```diff
@@ -531,6 +531,11 @@ periodics:
           # this is mostly for building kubernetes
           memory: 9Gi
           cpu: 7
+    tolerations:
```

Comment:
please don't move kind jobs, not running them in the same env makes debugging tricky and we want to make sure k/k presubmits don't break in particular

Reply:
I'm looking forward to moving these, but not during code freeze, and not as the first jobs

Reply:
also we have to be careful because moving to this pool implies cgroups v2, and we need to make sure not only does that not break but the kind repo will need to double check the GHA to make sure we retain cgroups v1 coverage

previously cgroups v1 = prow, cgroups v2 = GHA 

cc @stmcginnis @aojea